api/Element.json

```diff
@@ -6913,6 +7017,58 @@
           }
         }
       },
+      "show_event": {
+        "__compat": {
+          "description": "<code>show</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/show_event",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
```

Comment:
This is deprecated according to the page: https://developer.mozilla.org/en-US/docs/Web/API/Element/show_event